Adding more team member examples and repos (#4)
dfinity/awesome-dfinity: README.md

Submit a pull request to add a link to your repository here.

> ### Tungsten demo day

Reviewer: ```suggestion
### Demo day projects
```
I really recommend de-emphasizing the use of internal milestone names--they suggest multiple parallel networks and really only create confusion (for example, "tungsten" isn't different from"sodium" or "mercury").

- [Bonus](https://github.com/seniorjoinu/Bonus) - Next-generation loyalty system built on the Internet Computer.

> - [Get Impact Now](https://github.com/getimpactnow/getimpactnow/) - Digital platform which enables responsible citizens to be a part of the decision-making process.

Reviewer: ```suggestion
- [Get Impact Now](https://github.com/getimpactnow/getimpactnow/) - Digital platform that enables responsible citizens to be a part of the decision-making process.
```
Not sure if that's quite an accurate description of the app. "responsible" is sort of value judgement and I think it is more about "interested" citizens and the app enables them to see and interact on issues that are important to their communities (where in theory they have the most impact and in in the world--I'm not sure how to capture that idea of the concentric circles of influence/impact)

Author: For these, I just added the descriptions that were from the teams' GitHub repos themselves.

- [Internet Computer Queue System (ICQS)](https://github.com/davidp94/icqs-demo) - A bridge to legacy systems, such as email.

- [NNS Explorer](https://github.com/hashquark-io/nnsExplorer) - Mock explorer for the Internet Computer's Network Nervous System (NNS).

> - [Sailfish](https://github.com/sailfish-app) - The gateway to open financial services on the DFINITY Internet Computer.

Reviewer: ```suggestion
- [Sailfish](https://github.com/sailfish-app) - The gateway to open financial services on the Internet Computer.
```

- [Hero](https://github.com/stanleygjones/hackathon) - A random "killer app" generator inspired by the Wu-Tang Name Generator.

## Example apps from the DFINITY team

- [IC Game Terminal](https://github.com/matthewhammer/ic-game-terminal) - Keyboard input (⌨) and graphical output (📺) for programs on the Internet Computer.

- [Journey](https://github.com/hansl/journey) - An open blog engine written for the Internet Computer.

> - [Reversi](https://github.com/ninegua/reversi) - A multiplayer Reversi Game on Internet Computer.

Reviewer: ```suggestion
- [Reversi](https://github.com/ninegua/reversi) - A multiplayer Reversi game on Internet Computer.
```

> - [Superheroes](https://github.com/enzoh/superheroes) - A simple example that demonstrates how to build a CRUD application on the Internet Computer using Motoko and React.

Reviewer: ```suggestion
- [Superheroes](https://github.com/enzoh/superheroes) - A simple CRUD application that demonstrates using Motoko and React to create your own universe of superheroes that runs on the Internet Computer .
```
This one is a bit awkward. 

I might consider revising all of the descriptions to focus on what they do or what they illustrate.
Manage your own universe of superheroes with a simple application that illustrates how to create, read, update, and delete (CRUD) objects using Motoko andReact.

## Motoko resources

> - [Vessel package manager](https://github.com/kritzcreek/vessel)

Reviewer: ```suggestion
- [Vessel package manager](https://github.com/kritzcreek/vessel) - Add and manage packages, libraries, and dependencies for your Motoko programs.
```

## Motoko libraries

### Data structures

- [motoko-crud](https://github.com/matthewhammer/motoko-crud) - A generic CRUD framework for Motoko.

- [motoko-dht](https://github.com/enzoh/motoko-dht) - A Motoko Distributed Hash Table (DHT) canister.

- [motoko-graph](https://github.com/matthewhammer/motoko-graph) - Graphical data models for Motoko.

- [motoko-sequence](https://github.com/matthewhammer/motoko-sequence) - Cache-friendly, persistent sequential data for Motoko.

### Algorithms

- [mo-parsec](https://github.com/crusso/mo-parsec) - A [Parsec](https://hackage.haskell.org/package/parsec)-based parser combinator library for Motoko.

- [motoko-adapton](https://github.com/matthewhammer/motoko-adapton) - Dynamic dependence graph and memoization techniques in Motoko.

> - [motoko-bigsearch](https://github.com/matthewhammer/motoko-sequence/blob/master/service/BigSearch.mo) - A search service in Motoko, inspired by indexing and search systems like Apache Lucene.

Reviewer: Does this go here?

> - [motoko-redraw](https://github.com/matthewhammer/motoko-redraw) - 2D graphics and layout algorithms for Motoko.

Reviewer: Does this go here?

> ### Testing

Reviewer: ```suggestion
## Testing tools
```
